src/app/pages/server-compare/server-compare.component.html

```diff
@@ -24,12 +24,70 @@ <h1 class="text-white font-bold text-3xl">Server Comparison</h1>
           </button>
         </div>
       </div>
+      <div *ngIf="instances.length" class="text-white text-xl pr-16">
+        {{description}}
+      </div>
+    </div>
+
+    <div *ngIf="!instances.length" class="flex flex-col gap-4">
       <div class="text-white text-xl pr-16">
-        Compare cloud servers characteristics, such as CPU, GPU, memory and storage details, and the performance of the instances by various benchmarking workloads to find the optimal compute resource for your needs.
+        Server comparison helps you decide which server is the best for your needs. You can compare <a class="text-emerald-400" routerLink="/servers" >servers</a> based on their specifications, performance, and pricing.
       </div>
+      <div class="text-white text-xl pr-16">
+        Here is a list of notable or interesting compares between the servers.
+      </div>
+
+      <table class="items_table">
```

Comment:
this table might need some further styling to be more consistent with the other tables, e.g. striped background color, hover highlight and actionable rows